packages/prettier-config/index.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What about the [`quoteProps`](https://github.com/Shopify/quilt/blob/master/.prettierrc#L6) rule?

Reply:
Added.

Reply:
`"quoteProps": "as-needed"` and `"arrowParens": "always"` are prettier defaults (as of v2 at least). Do we need to specify those if they're already the default?s

Reply:
No, I guess those can be removed in that case.  Will test.